addons/x_lib/functions/actors/fnc_allActors.sqf

```diff
@@ -23,18 +23,21 @@ See Also:
     - <ALIVE_fnc_sendActorMessage>
 
 Author:
-    Naught
+    Naught, dixon13
 ---------------------------------------------------------------------------- */
 
-private ["_group"];
-_group = [_this, 1, ["GROUP"], ALiVE_actors_mainGroup] call ALiVE_fnc_param;
-
-if (!isNil "_group") then
-{
+params [["_group", grpNull]];
+
+if (_group == grpNull) then {
```

Comment:
`grpNull` can't be compared with the `==` operator, either use `isNull` or `isEqualTo`.